ExomeDepth/submit_batch_exomedepth.py

```diff
@@ -115,16 +113,26 @@ def process(bam, args, refset_dic):
         sys.exit("no bam files detected")
 
     refset_dic = {}
-    if args.refsetlist:
-        with open(args.refsetlist) as refset_file:
+    gender_dic = {}
+    warning_dic = {}
+    if args.reanalysis:
+        with open(args.reanalysis) as refset_file:
             for line in refset_file:
                 splitline = line.split()
                 if splitline[0] not in refset_dic:
                     refset_dic[splitline[0]] = splitline[1]
-
+                if len(splitline)>2:  #Note: make sure reanalysis file must contain sampleid in first column and refset in second column. More elegant way to program here?
+                    if settings.reanalysis_dic[splitline[2]]: 
+                        if splitline[0] not in gender_dic: 
+                            gender_dic[splitline[0]] = settings.reanalysis_dic[splitline[2]][0]
+                        if splitline[0] not in  warning_dic:
+                            warning_dic[splitline[0]] = settings.reanalysis_dic[splitline[2]][1]
+                    else: 
+                        sys.exit("Warning {} in reanalysis file does not exict".format(splitline[2]))
```

Comment:
I don't understand the sys.exit warning. You first check on line 124 whether reanalysis file contains at least 2 columns. If that is not the case you give a warning that the file does not exists. 

Secondly, create variables for splitline items. Very hard to follow what is precisely being done here.